src/containers/PreviewDraftPage/PreviewDraftPage.tsx

```diff
@@ -10,13 +10,26 @@ import { HelmetWithTracker } from '@ndla/tracker';
 import { Hero, HeroContentType, OneColumn } from '@ndla/ui';
 import { useTranslation } from 'react-i18next';
 import { useParams } from 'react-router-dom';
+import { spacing } from '@ndla/core';
+import { css } from '@emotion/react';
 import PreviewDraft from '../../components/PreviewDraft/PreviewDraft';
 import { getContentTypeFromResourceTypes } from '../../util/resourceHelpers';
 import { useTaxonomyVersion } from '../StructureVersion/TaxonomyVersionProvider';
 import LanguageSelector from './LanguageSelector';
 import { useDraft } from '../../modules/draft/draftQueries';
 import { useNodes } from '../../modules/nodes/nodeQueries';
 
+const frontpageStyling = css`
+  background-color: #f7fafd;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  margin: unset;
+  max-width: unset;
```

Comment:
Hvis vi skal unsette ting, trenger vi da egentlig å bruke OneColumn?

Reply:
Unngår conditional render da vi bruker denne for normale artikler også !